devicetypes/smartthings/aeotec-doorbell-siren-child.src/aeotec-doorbell-siren-child.groovy

```diff
@@ -20,15 +20,18 @@ metadata {
 
 	}
 	tiles {
-		multiAttributeTile(name: "chime", type: "generic", width: 6, height: 4) {
+		multiAttributeTile(name: "chime", type: "generic", width: 6, height: 2) {
 			tileAttribute("device.chime", key: "PRIMARY_CONTROL") {
 				attributeState "off", label: 'chime', action: 'chime.chime', icon: "st.alarm.alarm.alarm", backgroundColor: "#ffffff"
 				attributeState "chime", label: 'off', action: 'chime.off', icon: "st.alarm.alarm.alarm", backgroundColor: "#ff0000"
 			}
 		}
-		standardTile("off", "device.alarm", inactiveLabel: false, decoration: "flat") {
-			state "default", label: '', action: "alarm.off", icon: "st.secondary.off"
-		}
+		multiAttributeTile(name: "alarm", type: "generic", width: 6, height: 2) {
```

Comment:
Do two multi-attr tiles actually work??

Reply:
Yes, it does but there are two large buttons.

Reply:
Ah yes! So it does! I actually had never thought to try it. (Though I do see it in the docs, too.)